examples/custom-resource/controls/example.rb

```diff
@@ -0,0 +1,6 @@
+# encoding: utf-8
+
+describe gordon do
+  its('example') { should_not eq 'foobar' }
```

Comment:
My main concern is to use meaningful names.  Gordon here is meant to be Commissioner Gordon, from Batman.  What properties might he be dealing with?

its('crime_rate') { should be < 2 }
it { should have_a_fabulous_mustache }

I don't know what the 'example' property is, or why its value should or should not be  'foobar'.